yaksh/templates/yaksh/design_course_session.html

```diff
@@ -0,0 +1,110 @@
+{% extends "manage.html" %}
+{% load custom_filters %}
+{% block title %}Design Course Session{% endblock %}
+
+{% block pagetitle %}Design Course Session{% endblock %}
+
+{% block script %}
+<script src="{{ URL_ROOT }}/static/yaksh/js/jquery-1.9.1.min.js"></script>
+<script src="{{ URL_ROOT }}/static/yaksh/js/design_course.js"></script>
+{% endblock %}
+
+{% block css %}
+<link rel="stylesheet" media="all" type="text/css" href="{{ URL_ROOT }}/static/yaksh/css/design_course.css" />
+{% endblock %}
+
+{% block main %}
+<a href="{{URL_ROOT}}/exam/manage/courses/" class="btn btn-primary">
+    Back to Courses
+</a>
+<form action="{{URL_ROOT}}/exam/manage/courses/designcourse/{{course_id}}/" method="POST" id="design_course_form">
+{% csrf_token %}
+    <div class="tab-pane active" id="available-lesson-quiz">
+        <div class="row">
+            <div class="col-md-8 col-md-offset-2 available-list">
+                <div id="fixed-available-wrapper">
+                    <p><u><b>Available Lessons and quizzes: (Add Lessons and Quizzes)</b></u></p>
+                    <div id="fixed-available">
+                        <table id="course-details" class="table table-bordered">
+                        <tr>
+                        <th width="2%">Select</th>
+                        <th>Learning Module</th>
+                        <th>Learning Units</th>
```

Comment:
Can we please change this to "Lessons/Quizzes" instead of introducing new terminology